templates/english/topic-page.html

```diff
@@ -0,0 +1,475 @@
+                >
+                  Heading link
+                </gcds-link>
+              </gcds-heading>
+              <gcds-text>
+                Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.
+              </gcds-text>
+            </div>
+            <div>
+              <gcds-heading
+                margin-bottom="100"
+                margin-top="0"
+                tag="h3"
+              >
+                <gcds-link
+                  href="#"
+                  class="font-text"
+                >
+                  Heading link
+                </gcds-link>
+              </gcds-heading>
+              <gcds-text>
+                Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.
+              </gcds-text>
+            </div>
+            <div>
+              <gcds-heading
+                margin-bottom="100"
+                margin-top="0"
+                tag="h3"
+              >
+                <gcds-link
+                  href="#"
+                  class="font-text"
+                >
+                  Heading link
+                </gcds-link>
+              </gcds-heading>
+              <gcds-text>
+                Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.
+              </gcds-text>
+            </div>
+            <div>
+              <gcds-heading
+                margin-bottom="100"
+                margin-top="0"
+                tag="h3"
+              >
+                <gcds-link
+                  href="#"
+                  class="font-text"
+                >
+                  Heading link
+                </gcds-link>
+              </gcds-heading>
+              <gcds-text>
+                Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.
+              </gcds-text>
+            </div>
+            <div>
+              <gcds-heading
+                margin-bottom="100"
+                margin-top="0"
+                tag="h3"
+              >
+                <gcds-link
+                  href="#"
+                  class="font-text"
+                >
+                  Heading link
+                </gcds-link>
+              </gcds-heading>
+              <gcds-text>
+                Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.
+              </gcds-text>
+            </div>
+          </gcds-grid>
+        </section>
+
+        <!---------- Featured ---------->
+        <!-- TODO: Update the heading, image and text content. -->
+        <section>
+          <gcds-sr-only>
+            <gcds-heading margin-top="0" margin-bottom="0" tag="h2">Featured</gcds-heading>
```

Comment:
The `gcds-sr-only` component has a tag prop that can be set to `h2`. Would it be possible to use that here instead of adding the `gcds-heading`?